### ISSUE TITLE
Formatted output for breadcrumb data

### ISSUE THREAD

**Issue by tony** [getsentry/sentry#10221]
At the moment, to my knowledge, I don't think formatting exists for breadcrumbs?  To my knowledge, there are also no other issues on it.

![image](https://user-images.githubusercontent.com/26336/47322118-77ac2980-d61c-11e8-8573-325cf9a2935e.png)

At the moment, breadcrumbs don't seem to have formatting. This would be highly valuable to have.

![image](https://user-images.githubusercontent.com/26336/47322595-2309ae00-d61e-11e8-96b6-b56378915326.png)

In this case, being able to expand/collapse and check out nested data would be very valuable. The reason why there can be big spans of code where we may want to snapshot the state at a specific time.

The closest thing I've been able to find is context managers for adding extra context data: https://docs.sentry.io/learn/context/?platform=python#extra-context

Sort of related: https://github.com/getsentry/sentry/issues/5407

**Comment by StarryFire** [comment]
Yes, please! This would be nice to add! Its hard to read JSON data in breadcrumbs right now!

**Comment by BYK** [comment]
I think @priscilawebdev did something similar recently so assigning to her. @priscilawebdev LMK if this is something already done or planned to be done.

**Comment by tony** [comment]
@BYK @priscilawebdev Do we want to do anything with this issue? (It got 13 upvotes since I last checked!)

**Comment by priscilawebdev** [comment]
This is a great suggestion! ⭐️   I'm going to add this to my backlog and bring it up for discussion @BYK @tony

**Comment by priscilawebdev** [comment]
I will be working on this issue this week 😉

**Comment by tony** [comment]
@priscilawebdev Good to know!  Excited about this. This will make the output of our issues much cleaner!
